From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-cmans__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-cmans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

BryanLa committed:
updates from feedback

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates from PM feedback

BryanLa committed:
updates from PM feedback

Commit:
fix broken links in cosmos-db and devtest-lab and redirect json files

BryanLa committed:
final cleanup

BryanLa committed:
more refactoring and modifications

BryanLa committed:
more refactoring and modifications

BryanLa committed:
more refactoring and modifications

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-craic0202

Commit:
update links in more files

BryanLa committed:
Merge branch 'master' of https://www.github.com/microsoftdocs/azure-docs-pr into bryanla-dps-tutorialsetupdevice

ShawnJackson committed:
edit pass: marketplace-publishers-guide

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into marketplace-publishers-guide

ShawnJackson committed:
edit pass: marketplace-publishers-guide

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into marketplace-publishers-guide

ShawnJackson committed:
edit pass: marketplace-publishers-guide

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into marketplace-publishers-guide

ShawnJackson committed:
edit pass: marketplace-publishers-guide

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into marketplace-publishers-guide

ShawnJackson committed:
edit pass: marketplace-publishers-guide

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into marketplace-publishers-guide

ShawnJackson committed:
edit pass: marketplace-publishers-guide

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into marketplace-publishers-guide

ShawnJackson committed:
edit pass: marketplace-publishers-guide

Commit:
updates for the rebranding of OMS

Commit:
Merge branch 'master' into v-craic0320

Commit:
fix merge conflict

Commit:
update files after review

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-craic0320

Commit:
Merge branch 'v-craic0320' of https://github.com/craigcaseyMSFT/azure-docs-pr into v-craic0320

Commit:
Merge branch 'master' into v-craic0320

ShawnJackson committed:
edit pass: marketplace-publishers-guide

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into marketplace-publishers-guide

ShawnJackson committed:
Acrolinx and other fixes

Commit:
fixing link problem

Commit:
Merge branch 'v-craic0320' of https://github.com/craigcaseyMSFT/azure-docs-pr into v-craic0320

Commit:
fixing invalid link

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-craic0320

Commit:
Merge branch 'master' into v-craic0320

craiglandis committed:
Added Windows commands and known issue about debug mode

craiglandis committed:
Added Windows commands and known issue about debug mode take 2

craiglandis committed:
Added Windows commands and known issue about debug mode take 3

craiglandis committed:
Added Windows commands and known issue about debug mode take 4

neilpeterson committed:
[AKS] internal lb doc

neilpeterson committed:
boarder and toc

rmcmurray committed:
Updating the App Service Java Quickstart

rmcmurray committed:
Updating the App Service Java Quickstart

MGoedtel committed:
updated articles in support of new pricing plan

MGoedtel committed:
updated TOC

neilpeterson committed:
adding copy

Commit:
Re: [MicrosoftDocs/azure-docs] Using conda on py35 virtual environment (#5278)

MGoedtel committed:
removed backup file not intended to be included

MGoedtel committed:
corrected broken links

MGoedtel committed:
updated broken link

neilpeterson committed:
adding delete

neilpeterson committed:
adding next steps

craiglandis committed:
Added Windows commands and known issue about debug mode take 6

BryanLa committed:
Merge branch 'master' of https://www.github.com/microsoftdocs/azure-docs-pr into bryanla-dps-tutorialsetupdevice

Commit:
Task 1060: Update articles for using both caching rules and rules engine to enable DSA caching

craiglandis committed:
Added Windows commands and known issue about debug mode take 7

craiglandis committed:
Added Windows commands and known issue about debug mode take 8

Commit:
Task 1060: Edits

neilpeterson committed:
adding static ip

neilpeterson committed:
adding prompt to code blocks

MGoedtel committed:
Updated content per review and PM feedback

MGoedtel committed:
updated manage cost article

Commit:
fixing links for CLI topics

ggailey777 committed:
Remove favoriting instructions from the quickstarts

As verified in A-B test experiment
[cc996988-fb4f-47](http://skyeyereports.cloudapp.net/experiment?id=cc996988-fb4f-47)

MGoedtel committed:
updated intro paragraph to cost mgmt article

MGoedtel committed:
Updated H1

tamram committed:
update for public PR 4378

ggailey777 committed:
Fixed link

ggailey777 committed:
Remove old favorite include

tamram committed:
updating list of RP options

elisolMS committed:
updating location of tenant creation doc in TOC

cherylmc committed:
new

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into create2

eringreenlee committed:
fixed networking and aadds105

mimig1 committed:
Cosmos DB: Correcting TOC mismatches

mimig1 committed:
Removed duplicate distribute-data-globally instances

zhiweiwangmsft committed:
Update Autdit logs notice

v-geberr committed:
tutorial composite entity

Commit:
Add reference

v-geberr committed:
edits

KumudD committed:
created redirects

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into createredirects

v-geberr committed:
edits

bwren committed:
index

mimig1 committed:
adding the index page to the PR

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 0328-composite-entity-tutorial

v-geberr committed:
toc

BryanLa committed:
updates from feedback

KumudD committed:
removed files

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into createredirects

mimig1 committed:
Cosmos DB: Corrected links to Cosmos DB forum

curtand committed:
[Azure AD users] warning that after 30 days, deletion is permanent

ktoliver committed:
Merge pull request #36490 from ggailey777/experiment

Remove favoriting instructions from the quickstarts

ramankumarlive committed:
updates for Managed Disks 

updates for Managed Disks

GitHubber17 committed:
Merge pull request #36502 from cherylmc/create2

CLI

curtand committed:
made it  WARNING

v-geberr committed:
links to quickstarts and tutorials in how-to section

GitHubber17 committed:
Merge pull request #36509 from mimig1/03282018-tocfixes

Cosmos DB: Correcting TOC mismatches

mimckitt committed:
Fixes error in Code Example

The following line shows: 

New-AzureRmServiceFabricCluster -ResourceGroupName $resourceGroup -SecretIdentifier $secretId -TemplateFile $templateFilePath -ParameterFile $parameterFilePath 

When calling the resource group name variable it should be $resourceGroupName but in this example it was incorrect and stated $resourceGroup

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#6414

Add link to quota request doc

PRMerger15 committed:
Merge pull request #36531 from curtand/restore0328

[Azure AD users] warning that after 30 days, deletion is permanent

GitHubber17 committed:
Merge pull request #36525 from KumudD/createredirects

Redirect for internal and public Load Balancer Overview topics

GitHubber17 committed:
Merge pull request #31814 from BryanLa/bryanla-dps-tutorialsetupdevice

Tutorial: Set up device - updates from review/feedback

v-geberr committed:
edits

GitHubber17 committed:
Merge pull request #36431 from dksimpson/dean-950

Task 1060: Update articles for using both caching rules and rules eng…

PRMerger18 committed:
Merge pull request #36535 from iainfoulds/patch-4

[VM] Fixes MicrosoftDocs/azure-docs#6414

PRMerger18 committed:
Merge pull request #36534 from MicahMcKittrick-MSFT/patch-10

Fixes MicrosoftDocs/azure-docs#6437

markgalioto committed:
adding link to back up VM template based on the customer request

ktoliver committed:
fix broken bold markdown

fix broken bold markdown

v-geberr committed:
added other how-to pages

Commit:
Merge pull request #36532 from ramankumarlive/patch-4

updates for Managed Disks

ktoliver committed:
Merge pull request #36513 from v-geberr/0328-composite-entity-tutorial

tutorial composite entity

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 0328-concept-entities

v-geberr committed:
edits for composite tutorial

v-geberr committed:
edits

ShawnJackson committed:
Update marketplace-publishers-guide.md

v-geberr committed:
add new tutorial to landing page and correct spelling of word in title

dlepow committed:
N-series: spec updates

kpiteira committed:
Added retirement message to all pages

markgalioto committed:
editing the link to template

v-geberr committed:
edits

Juliako committed:
Update media-services-hyperlapse-content.md

BryanLa committed:
Fix owner

BryanLa committed:
Fix owner

klaaslanghout committed:
adding registered servers scale limit

kpiteira committed:
Merge branch 'master' into azme-retire

PRMerger19 committed:
Merge pull request #36544 from Juliako/patch-8

Update media-services-hyperlapse-content.md

markgalioto committed:
another attempt to link out to the Templates

Commit:
Merge pull request #36545 from BryanLa/patch-20

Fix owner

Commit:
Merge pull request #36547 from BryanLa/patch-21

Fix owner

tamram committed:
Update storage-decide-blobs-files-disks.md

tamram committed:
Update storage-decide-blobs-files-disks.md

bwren committed:
redirection

PRMerger13 committed:
Merge pull request #36538 from markgalioto/mar282018

adding link to back up VM template based on the customer request

tracsman committed:
Fix ER FAQ Typo

tamram committed:
Update storage-decide-blobs-files-disks.md

tamram committed:
updating storage account limits to 200 per region

Commit:
Merge pull request #36552 from tracsman/jonorTemp

Fix ER FAQ Typo

genlin committed:
add link

rboucher committed:
updated all screen shots. Minor text corrections

craiglandis committed:
Added Windows commands and known issue about debug mode take 9

genlin committed:
correct typo

rboucher committed:
fix broken link

GitHubber17 committed:
Merge pull request #36541 from v-geberr/0328-landing-page-composite-tutorial

add new tutorial to landing page and correct spelling of word in title

GitHubber17 committed:
Merge pull request #36334 from craiglandis/cl-serial-console-update1

Added Windows commands and known issue about debug mode

GitHubber17 committed:
Merge pull request #36488 from SyntaxC4-MSFT/cfowler-cli-link-fix

[App Service] Update CLI Links for Samples

GitHubber17 committed:
Merge pull request #36500 from elisolMS/move-tenant-doc-in-TOC

updating location of tenant creation doc in TOC

PRMerger14 committed:
Merge pull request #36557 from genlin/master304

vpn-gateway-troubleshoot-site-to-site-cannot-connect.md

tdykstra committed:
note that df works in 1.x and 2.x

PRMerger19 committed:
Merge pull request #36563 from tdykstra/df

note that df works in 1.x and 2.x

GitHubber17 committed:
Merge pull request #36546 from kpiteira/azme-retire

Added retirement message to all pages

GitHubber17 committed:
Merge pull request #36548 from klaaslanghout/patch-21

adding registered servers scale limit

msvijayn committed:
Correction of text

Text corrections

GitHubber17 committed:
Merge pull request #36526 from bwren/monitoring-landingpage

Monitoring landing page

GitHubber17 committed:
Merge pull request #36551 from MicrosoftDocs/tamram-patch-4

Update storage-decide-blobs-files-disks.md

GitHubber17 committed:
Merge pull request #36558 from rboucher/gartnerfix

updated all screen shots. Minor text corrections

PRMerger17 committed:
Merge pull request #36492 from tamram/tamram-0328

update for public PR 4378

PRMerger20 committed:
Merge pull request #36530 from eringreenlee/master

fixed networking and aadds105

PRMerger20 committed:
Merge pull request #36533 from v-geberr/0328-concept-entities

links to quickstarts and tutorials in how-to section

PRMerger20 committed:
Merge pull request #36512 from zhiweiwangmsft/patch-27

Update Autdit logs notice

PRMerger19 committed:
Merge pull request #36567 from msvijayn/gartner-vijayn

Correction of text

rwike77 committed:
Added Windows standalone cluster links

PRMerger20 committed:
Merge pull request #36543 from dlepow/fix

N-series: correction

tjvishnu committed:
Updated docs based on new features and more FAQs

Added more documentation on additional questions from customer interactions and CSS queries. Also updated to reflect public preview availability of Backup for Azure File Sync

Juliako committed:
Update use-oauth2-for-authorization.md

Juliako committed:
Update use-oauth2-for-authorization.md

tjvishnu committed:
Updated some more

Commit:
Merge pull request #36572 from Juliako/patch-8

Update use-oauth2-for-authorization.md

ajaycode committed:
added region - South UK

PRMerger19 committed:
Merge pull request #36576 from ajaycode/network-monitoring-overview

added region - South UK

v-geberr committed:
remove fuzzy search

v-geberr committed:
remove fuzzy search

vhorne committed:
update FAQ per Amit

vhorne committed:
acrofix

MGoedtel committed:
updated usage article

Commit:
Merge pull request #36580 from vhorne/http2-upd

Http2 upd

v-geberr committed:
regex home page changes

v-geberr committed:
edit

itechedit committed:
Merge pull request #36554 from tamram/tamram-0328a

updating storage account limits to 200 per region

Commit:
fix conflicts and updates based on review

Commit:
Merge pull request #36581 from v-geberr/0329-regex-update-1

0329 regex update 1

bwren committed:
remove oms - solution resources

Commit:
Merge branch 'v-craic0320' of https://github.com/craigcaseyMSFT/azure-docs-pr into v-craic0320

itechedit committed:
Merge pull request #36571 from tjvishnu/patch-2

Updated docs based on new features and more FAQs

GitHubber17 committed:
Merge pull request #36344 from rmcmurray/20180326

Updating the App Service Java Quickstart

MGoedtel committed:
updated paragraph

itechedit committed:
Merge pull request #36577 from v-geberr/0329-remove-fuzzy-search-from-images

remove fuzzy search

itechedit committed:
Merge pull request #36578 from v-geberr/0329-remove-fuzzy-search-2

remove fuzzy search

MGoedtel committed:
removed images

MGoedtel committed:
updated images

PRMerger17 committed:
Merge pull request #35841 from craigcaseyMSFT/v-craic0320

updates for the rebranding of OMS

PRMerger18 committed:
Merge pull request #36371 from cjgronlund/what-is-fest

Re: [MicrosoftDocs/azure-docs] Using conda on py35 virtual environmen…

neilpeterson committed:
updating a few words / Marsh review

brapel committed:
bing-custom-search: fix numbering and formatting

curtand committed:
[Azure AD groups] add license type

PRMerger13 committed:
Merge pull request #36583 from bwren/monitoring-solutions

OMS Retire - solution resources

Ja-Dunn committed:
Update marketplace-publishers-guide.md

punctuation

MGoedtel committed:
updated articles per editor feedback

PRMerger15 committed:
Merge pull request #36593 from curtand/kairaz0329

[Azure AD groups] add license type

PRMerger15 committed:
Merge pull request #36589 from brapel/bing-custom-search-minor-updates

bing-custom-search: fix numbering and formatting

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

GitHubber17 committed:
Merge pull request #36340 from neilpeterson/aks-internal-lb

[AKS] internal lb doc

PRMerger16 committed:
Merge pull request #36528 from mimig1/patch-234

Cosmos DB: Corrected links to Cosmos DB forum

GitHubber17 committed:
Update log-analytics-usage.md

Commit:
retarget link from REST API to current valid CLI command

Ja-Dunn committed:
Update marketplace-publishers-guide.md

capping

rmca14 committed:
Merge pull request #32914 from craigcaseyMSFT/v-craic0202

fix broken links in cosmos-db and devtest-lab and redirect json files

v-alje committed:
Merge pull request #36597 from MicrosoftDocs/FromPublicRepo

From public repo

GitHubber17 committed:
Merge pull request #36368 from MGoedtel/UpdateLAPricing327

updated articles in support of new pricing plan

Commit:
Merge pull request #36598 from stevestein/sql-db-connectivity-article

retarget link from REST API to current valid CLI command

Ja-Dunn committed:
Merge pull request #35966 from ShawnJackson/marketplace-publishers-guide

edit pass: marketplace-publishers-guide

Commit:
Merge pull request #36570 from rwike77/domainaccounts

Added Windows standalone cluster links
